main.py

```diff
@@ -0,0 +1,115 @@
+from typing import Tuple, List
+
+from dulwich.repo import Repo
+from dulwich.walk import WalkEntry
+from dulwich.diff_tree import TreeChange
+import difflib
+import json
+
+
+def calc_diff(repo: Repo, change: TreeChange) -> Tuple[int, int]:
+    """
+    get amount of added and deleted lines
+
+    :param repo: repository with change
+    :param change: change of file
+    :return: added number, deleted number
+    """
+    d = difflib.Differ()
+    diffs = d.compare(repo.get_object(change.old.sha).data.decode().splitlines(),
+                      repo.get_object(change.new.sha).data.decode().splitlines())
+
+    added = 0
+    deleted = 0
+    for j in diffs:
+        if j[0] == '+' and j[1] != '+':
+            added += 1
+        elif j[0] == '-' and j[1] != '-':
+            deleted += 1
+
+    return added, deleted
+
+
+def get_diff(repo: Repo, entry: WalkEntry) -> dict:
+    """
+    get information about commit such as:
+        number of added lines for each file
+        number of deleted lines for each file
+        blob_id for each file
+
+    :param repo: pending repository
+    :param entry: commit object
+    :return: info in form of dict {TODO}
+    """
+    res = {}

```

Comment:
let's use `defaultdict`
```suggestion
    res = defaultdict(dict)
```

so you can do
```python
res[some_key][another_key] = some_value
```